modules/ROOT/pages/dw-crypto-functions-hmacbinary.adoc

```diff
@@ -35,13 +35,13 @@ This example uses HMAC with a secret value to encrypt the input content.
 import dw::Crypto
 output application/json
 ---
-{ "HMACBinary" : Crypto::HMACBinary("key_re_loca" as Binary, "xxxxx" as Binary) }
+{ "HMACBinary" : Crypto::HMACBinary("xxxxx" as Binary, "key_re_loca" as Binary) }
 ----
 
 ==== Output
 
 [source,JSON,linenums]
 ----
-{ "HMACBinary": ".-\ufffd\ufffd\u0012\ufffdۊ\ufffd\ufffd\u0000\ufffd\u0012\u0018R\ufffd\ufffd=\ufffd*" }
+{ "HMACBinary": "\ufffd@1\ufffd\u001f\ufffdA\ufffdX\u0006\ufffd-\u001c\ufffd\ufffd\ufffd\ufffd\u0004\ufffd\ufffd" }
```

Comment:
The change itself is fine, but we need to fix this example. Probably after @machaval adds the new Binary literals